cluster/vsphere/config-default.sh

```diff
@@ -34,6 +34,9 @@ NODE_CPU=1
 
 SERVICE_CLUSTER_IP_RANGE="10.244.240.0/20"  # formerly PORTAL_NET
 
+# Admission Controllers to invoke prior to persisting objects in cluster
+ export ADMISSION_CONTROL="NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,SecurityContextDeny"
```

Comment:
Sure. Added.

Yes agree that this shall be first get from probably environment, then if
not present set to default.

On Mon, Apr 25, 2016 at 9:31 PM, Brendan Burns <notifications@github.com>
wrote:

> In cluster/vsphere/config-default.sh
> <https://github.com/kubernetes/kubernetes/pull/24201#discussion_r61027598>
> :
>
> > @@ -34,6 +34,9 @@ NODE_CPU=1
> >
> >  SERVICE_CLUSTER_IP_RANGE="10.244.240.0/20"  # formerly PORTAL_NET
> >
> > +# Admission Controllers to invoke prior to persisting objects in cluster
> > + export ADMISSION_CONTROL="NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,SecurityContextDeny"
>
> ok, can we add that for consistency (really we should probably get it in a
> global variable somewhere...)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24201/files/0d1904ff16037dbbb8c50797120093f73d763a99#r61027598>
>